cgp/cartesian_graph.py

```diff
@@ -173,7 +173,7 @@ def _determine_active_nodes(self) -> Dict[int, Set[Node]]:
                 continue
             active_nodes_by_hidden_column_idx[self._hidden_column_idx(node.idx)].add(node)
 
-            for i in node.inputs:
+            for i in node.input_nodes:
```

Comment:
when reading this line, i would prefer something like `for input_idx in node.input_node_indices`, i.e., renaming `input_nodes` -> `input_node_indices` or this overdoing it? ;)

Reply:
I think this is overdoing it. ;-)